x/avstask/types/expected_keepers.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// AccountKeeper defines the expected account keeper used for simulations (noalias)
+type AccountKeeper interface {
```

Comment:
Why expected these two keepers? Seemed no use case in the pr

Reply:
fixed